scripts/gke/destroy-old-clusters.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
go?

Reply:
cloudtest use bash commands to create,destroy and control gke cloud. So bash scripts is the simpliest way to find and destroy old clusters